src/keyfile.c

```diff
@@ -777,6 +781,8 @@ static void load_dialog_prefs(GKeyFile *config)
 	file_prefs.tab_order_ltr = utils_get_setting_boolean(config, PACKAGE, "tab_order_ltr", TRUE);
 	file_prefs.tab_order_beside = utils_get_setting_boolean(config, PACKAGE, "tab_order_beside", FALSE);
 	interface_prefs.show_notebook_tabs = utils_get_setting_boolean(config, PACKAGE, "show_notebook_tabs", TRUE);
+	if (interface_prefs.auto_sort_tabs_filename && interface_prefs.auto_sort_tabs_pathname)
+		interface_prefs.auto_sort_tabs_filename = FALSE;
```

Comment:
is this useful? also, it should probably go some place else, so that it's executed when loading settings directly rather than when displaying them, shouldn't it?